src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.ConvertValueToParamRewriter.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp.ReplacePropertyWithMethods
+{
+    internal partial class CSharpReplacePropertyWithMethodsService
+    {
+        private class ConvertValueToParamRewriter : CSharpSyntaxRewriter
+        {
+            public static readonly CSharpSyntaxRewriter Instance = new ConvertValueToParamRewriter();
+
+            private ConvertValueToParamRewriter()
+            {
+            }
+
+            private XmlNameSyntax ConvertToParam(XmlNameSyntax name)
+                => name.ReplaceToken(name.LocalName, SyntaxFactory.Identifier("param"));
+
+            private static bool IsValueName(XmlNameSyntax name)
+                => name.Prefix == null &&
+                   name.LocalName.ValueText == "value";
+
+            public override SyntaxNode VisitXmlElementStartTag(XmlElementStartTagSyntax node)
+            {
+                if (!IsValueName(node.Name))
+                    return base.VisitXmlElementStartTag(node);
+
+                return node.ReplaceNode(node.Name, ConvertToParam(node.Name))
+                    .AddAttributes(SyntaxFactory.XmlNameAttribute("value"));
+            }
```

Comment:
I was going to if it (the longer `return` statement's expression) fit on one line, but the additional call to `AddAttributes` seemed to pass a complexity line where the "light" syntax wasn't feeling so light anymore.